mLRS/Common/hal/stm32/rx-hal-WioE5-Mini-wle5jc.h

```diff
@@ -189,6 +190,26 @@ void led_red_on(void) { gpio_low(LED_RED); }
 void led_red_toggle(void) { gpio_toggle(LED_RED); }
 
 
+//-- SystemBootLoader
+
+#define BOOT_BUTTON               IO_PB13
+
+extern "C" { void delay_ms(uint16_t ms); }
+
+void systembootloader_init(void)
+{
+    gpio_init(BOOT_BUTTON, IO_MODE_INPUT_PU, IO_SPEED_DEFAULT);
+    delay_ms(10);
```

Comment:
is delay needed?

on the E77 kit it is needed since there is a capacitor which needas time to load. Here too?
if not, remove pl (see various R9 or FRM303)
if yes, pl add comment as in E77 kit

Reply:
No capacitor, just 470 resistor to ground.  So removed.